### ISSUE TITLE
Auth guard driver [api] is not defined.

### ISSUE THREAD

**Issue by estebangarcia** [dingo/api#958]
Hi,

I'm using Lumen 5.2 and I'm not able to use JWT Authentication I'm getting this error:

`Auth guard driver [api] is not defined.`

This is my code: https://github.com/estebangarcia/lumen_dingo_jwt

If anyone can point me in the right direction I'll really appreciate it

Thanks

**Comment by wemersonjanuario** [comment]
I have this same problem.

**Comment by estebangarcia** [comment]
No issues when using Laravel though.

**Comment by jasonlewis** [comment]
What is this?

```php
'auth' => [
    'jwt' => Dingo\Api\Auth\Provider\JWT::class,
]
```

That's in your `api.php` config. That's not correct.

Also, I think 5.2 changed how config files are loaded. You'll need to copy the entire `auth.php` file from the default location and configure it manually with `$app->configure('auth')` in your `bootstrap/app.php` file.

**Comment by estebangarcia** [comment]
For those experiencing the same issue. I was able to fix it in Lumen 5.2 by adding the auth.php file from Laravel, there is a working example in the first comment.

Thanks @jasonlewis for you guidance.

**Comment by phamduong** [comment]
@estebangarcia 
I see you add
`'api' => [
            'driver' => 'jwt',
            'provider' => 'users',
        ],`

But in **AuthManager**  class we don't have function: createJwtDriver
How does it work?
